app/components/candidate_interface/continuous_applications/application_submit_component.html.erb

```diff
@@ -10,21 +10,14 @@
   <%= form.govuk_submit t('continue') %>
 <% else %>
   <% errors.each do |error| %>
-    <p class="govuk-body">
-      <% if error.type == :incomplete_details %>
-        <p class="govuk-body">You need to <%= govuk_link_to('complete your details', candidate_interface_continuous_applications_details_path) %> before you can submit this application.</p>
-        <p class="govuk-body">This application will be saved as a draft while you finish your details.</p>
-      <% elsif error.type == :immigration_status %>
+    <% error.message.split(/\n+/).each_with_index do |line, idx| %>
+      <% if idx == 0 && error.type == :immigration_status %>
```

Comment:
Minor but can we renamed the var to `index`, please? and `index.zero?`

Reply:
Sure